deployer/templates/hawkular-cassandra-node-pv.yaml

```diff
@@ -28,6 +28,14 @@ parameters:
 - description: The size of storage requested.
   name: PV_SIZE
   value: 10Gi
+- description: Path within the container to the file containing the Certificate Authorities to add to the peer trust store
+  name: TRUSTSTORE_NODES_AUTHORITIES
+  value: '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt'
+  required: false
+- description: Path within the container to the file containing the Certificate Authorities to add to the client trust store
+  name: TRUSTSTORE_CLIENT_AUTHORITIES
+  value: '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt'
+  required: false
```

Comment:
So, the default should be `/hawkular-cassandra-certs/tls.client.truststore.crt` ? Note that we are overriding this on the deployer and the value on this template is just the "default" one.